core/src/edu/cornell/gdiac/temporary/entity/CalibNote.java

```diff
@@ -0,0 +1,187 @@
+package edu.cornell.gdiac.temporary.entity;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.math.Vector2;
+import edu.cornell.gdiac.temporary.GameCanvas;
+import edu.cornell.gdiac.util.FilmStrip;
+
+public class CalibNote {
+
+    /** How fast we change frames (one frame per 4 calls to update) */
+    private static final float ANIMATION_SPEED = 0.25f;
+    /** The number of animation frames in our filmstrip */
+    private static final int   NUM_ANIM_FRAMES = 4;
+    /** Current animation frame for this shell */
+    private float animeframe;
+    // Attributes for all game objects
+    /** Object position (centered on the texture middle) */
+    protected Vector2 position;
+    /** Object velocity vector */
+    protected Vector2 velocity;
+    /** Reference to texture origin */
+    protected Vector2 origin;
+    /** Radius of the object (used for collisions) */
+    protected float radius;
+    /** CURRENT image for this object. May change over time. */
+    protected FilmStrip animator;
+    private static final float SHELL_SIZE_MULTIPLE = 4.0f;
+    /** How fast we change frames (one frame per 4 calls to update) */
+
+    /**
+     * Constructor for a new CalibNote
+     */
+    public CalibNote() {
+        velocity = new Vector2();
+        position = new Vector2();
+    }
+
+    public void setTexture(Texture texture) {
+        animator = new FilmStrip(texture,1,NUM_ANIM_FRAMES,NUM_ANIM_FRAMES);
+        origin = new Vector2(animator.getRegionWidth()/2.0f, animator.getRegionHeight()/2.0f);
+        radius = animator.getRegionHeight() /  2.0f;
+    }
+
+    /**
+     * Returns the position of this object (e.g. location of the center pixel)
+     *
+     * The value returned is a reference to the position vector, which may be
+     * modified freely.
+     *
+     * @return the position of this object
+     */
+    public Vector2 getPosition() {
```

Comment:
i think gameobject (or some kind of resemblance) is necessary so we dont need these basic getters setters. this is a todo